### PR TITLE
Clarifying the driver section

### PR DESCRIPTION
**Pull request by crahrig**
Added a link to the preview driver steps needed for tf-dml to work with WSL and renamed the third section.

### CODE REVIEW

File: tensorflow/docs/Troubleshooting-Timeouts.md [hunk 1]
```diff
@@ -28,13 +28,15 @@ pip install tensorflow-directml --upgrade
 
 ### #2: Ensure your GPU drivers are up-to-date
 
-We recommend using the latest stable drivers from your GPU vendor wherever possible. GPU driver updates can be downloaded from the following locations:
+When running tensorflow-directml on native Windows we recommend using the latest stable drivers from your GPU vendor wherever possible. GPU driver updates can be downloaded from the following locations:
```
Comment (jstoecker) [inline tensorflow/docs/Troubleshooting-Timeouts.md]: I would leave this bit off, as the Windows driver installers include both native Windows and WSL UMDs (if they don't already for all IHVs, they will at some point).

File: tensorflow/docs/Troubleshooting-Timeouts.md [hunk 2]
```diff
@@ -28,13 +28,15 @@ pip install tensorflow-directml --upgrade
...
 
 * **For AMD GPUs:** [AMD Drivers and Support](https://www.amd.com/en/support)
 * **For Intel GPUs:** [Intel Download Center](https://downloadcenter.intel.com/)
 * **For NVIDIA GPUs:** [NVIDIA Driver Downloads](https://www.nvidia.com/Download/index.aspx)
 
-### #3: Exclude smaller GPUs from being used with DirectML
+If running tensorflow-directml inside of the Windows Subsystem for Linux (WSL) follow the [Enable TensorFlow with DirectML in WSL 2](https://docs.microsoft.com/en-us/windows/win32/direct3d12/gpu-tensorflow-wsl#install-the-preview-gpu-driver) guidance to install the approraite drivers.
```
Comment (jstoecker) [inline tensorflow/docs/Troubleshooting-Timeouts.md]: Typo

```suggestion
If running tensorflow-directml inside of the Windows Subsystem for Linux (WSL) follow the [Enable TensorFlow with DirectML in WSL 2](https://docs.microsoft.com/en-us/windows/win32/direct3d12/gpu-tensorflow-wsl#install-the-preview-gpu-driver) guidance to install the appropriate drivers.
```

**Review by jstoecker — CHANGES_REQUESTED** [review]

File: tensorflow/docs/Troubleshooting-Timeouts.md (hunk 1 above)
Comment (crahrig) [reply tensorflow/docs/Troubleshooting-Timeouts.md]: Makes sense, removed this part in update.

File: tensorflow/docs/Troubleshooting-Timeouts.md (hunk 2 above)
Comment (crahrig) [reply tensorflow/docs/Troubleshooting-Timeouts.md]: Thanks for this catch.

**Review by adtsai — APPROVED** [review]

**Review by jstoecker — APPROVED** [review]

**Pull request merged**